plugins/modules/zos_copy.py

```diff
@@ -1705,33 +1705,56 @@ def copy_to_pdse(
         existing_members = datasets.list_members(dest)  # fyi - this list includes aliases
         overwritten_members = []
         new_members = []
+        bulk_src_members = ""
+        result = dict()
 
         for src_member, destination_member in zip(src_members, dest_members):
             if destination_member in existing_members:
                 overwritten_members.append(destination_member)
             else:
                 new_members.append(destination_member)
-
+            bulk_src_members += "{0} ".format(src_member)
+
+        # Copy section
+        if src_ds_type == "USS" or self.asa_text or len(src_members) == 1:
+            """
+            USS -> MVS : Was kept on member by member basis bc file names longer that 8
```

Comment:
Your comment made me think of something, this is a good point **"file names longer that 8 characters will throw an error when copying to a PDS, because of the member name character limit."** , do we need an issue to address this, we could document that when we encounter names longer than 8 going to a PDS/E we will truncate them to 8, this is probably more common in a `dir` to PDSE copy where they did not carefully inspect the file names in a directory or just let it error. 

There is a `is_compatible` code section we could fit this in the future.

Reply:
Current behavior when copying a dir to a PDS/E is to truncate to the first 8 characters and remove any dots found(maybe we can explore what happens with files including special char, it may fail). We could add into the `src` section :
```
If src is USS, file names will be truncated and/or modified to ensure a valid name for a data set or member.
```
But that is kind of obvious?

Also, about documenting copy between MVS -> PDS/E being faster, we could add into `asa_text` section:
```
Using asa_text will result in a slower copy performance when copying multiple members.
```

What do you think?